modules/ROOT/pages/audit-log-collection.adoc

```diff
@@ -0,0 +1,108 @@
+= Audit log collection
+
+With just a simple configuration change, you can configure the PDPs to securely send audit logs to Cerbos Hub. This vastly simplifies the work that would otherwise be required to configure and deploy a log aggregation solution to securely collect, store and query audit logs from across your fleet.
+
+To get started, you need to obtain a set of client credentials. Navigate to the Settings tab of your workspace and click on "Generate client credentials" to generate a client ID and a client secret. The client secret is not stored on Cerbos Hub and cannot be recovered. Make sure to save it in a secure place.
+
+The client credentials can be provided to the PDP using environment variables or the configuration file. The environment variables to set are:
+
+[horizontal]
+`CERBOS_HUB_CLIENT_ID`:: Client ID
+`CERBOS_HUB_CLIENT_SECRET`:: Client secret
+`CERBOS_HUB_PDP_ID`:: Optional. A unique name for the PDP. If not provided, a random value is used.
+
+Alternatively, you can define these values in the Cerbos configuration file as follows:
+
+[source,yaml]
+----
+hub:
+  credentials:
+    pdpID: "..." # Optional. Identifier for this Cerbos instance.
+    clientID: "..." # ClientID
+    clientSecret: "..." # ClientSecret
+----
+
+To enable audit log collection, configure the `hub` audit log backend with a local storage path. This local storage path is important for preserving the audit logs until they are safely saved to Cerbos Hub. If there are any network interruptions or if the PDP process crashes, the audit logs generated up to that point are saved on disk and will be sent to Cerbos Hub the next time the PDP starts. If you're using a container orchestrator or a cloud-based solution to deploy Cerbos, attach a persistent storage volume at this path to ensure that the data does not get lost.
+
+[source,yaml]
+----
+server:
+  httpListenAddr: ":3592" # The port the HTTP server will listen on
+  grpcListenAddr: ":3593" # The port the gRPC server will listen on
+
+hub:
+  credentials:
+    pdpID: "..." # Optional. Identifier for this Cerbos instance.
+    clientID: "..." # ClientID
+    clientSecret: "..." # ClientSecret
+
+audit:
```

Comment:
is there any way we could make a default config for this which would allow someone just to set `CERBOS_HUB_LOGS=true` or such and negate the need to create and mount a config in their container?

Reply:
They can do `--set=audit.backend=hub --set=audit.hub.storagePath=/path`. Setting the storage path is important and I want the users to think about it. Otherwise it has the potential for blowing up.

Reply:
I agree setting the storage path is important for when you goto production, but I want to make it as simple as possible for someone just kicking the tyres on this. I always get confused about where/how to pass in arguments to a container and have to look it up.

Reply:
I'll document the commandline args method for reference.

I did think about this before but I decided not to do it for a couple of reasons.

- Having settings that are not suitable for production, especially when they are env vars, is a foot gun. While it's technically not our fault, it's a bad experience overall -- particularly when it has the potential for losing data. A lot of prototypes tend to simply become production when they "just work" as well.
- I think it's important that products don't give a false impression about the work it takes to run in production.